docs/admin/setup.mdx

```diff
@@ -87,7 +87,7 @@ The following options may be used to control what is generated:
 - `--hostname-prefix` is the prefix for node host names e.g. node results in node0, node1, etc.
 - `--hostname-suffix` is the suffix for node host names e.g. .example.com results in node0.example.com, node1.example.com, etc.
 - `--starting-ip` is the starting IP address of the first network node, with subsequent nodes on the next address in the subnet
-- `--hostname` overrides all hostnames of the nodes
+- `--hostnames` overrides all hostnames of the nodes
```

Comment:
I've updated the description to "override all hostnames of the nodes (list of hostnames must be the same length as the number of nodes)" in https://github.com/kwilteam/kwil-db/pull/373/files